unity/CAVE/Assets/Resources/Materials/Fonts/Arial SDF.asset

```diff
@@ -70,9 +70,9 @@ Material:
     - _OutlineWidth: 0
     - _PerspectiveFilter: 0.875
     - _Reflectivity: 10
-    - _ScaleRatioA: 1
-    - _ScaleRatioB: 1
-    - _ScaleRatioC: 1
+    - _ScaleRatioA: 0.8333333
+    - _ScaleRatioB: 0.6770833
+    - _ScaleRatioC: 0.6770833
```

Comment:
What's up with this? Were the fonts looking weird?

Reply:
There were two font files that had the same scale ratio values, they were the two that I actually I added to Unity manually. Very interesting. I'll set them back to 1 and if I run into issues I'll set them back